Content.Client/MedicalScanner/UI/MedicalScannerWindow.xaml.cs

```diff
@@ -38,7 +39,12 @@ public void Populate(MedicalScannerBoundUserInterfaceState state)
             {
                 text.Append($"{Loc.GetString("medical-scanner-window-entity-health-text", ("entityName", entity.Name))}\n");
 
-                var totalDamage = state.DamagePerType.Values.Sum();
+                var totalDamage = FixedPoint2.Zero;
+                foreach (var val in state.DamagePerType.Values)
+                {
+                    totalDamage += val;
+                }
+                
```

Comment:
Lack of .Sum() feels a little iffy, though I believe (as of writing this) this is the only place that uses it.

Reply:
there's two places that use it. i tried to fix it but could not figure out with C#